src/w3c/headers.js

```diff
@@ -108,6 +107,14 @@ const W3CDate = new Intl.DateTimeFormat(["en-AU"], {
   day: "2-digit",
 });
 
+/**
+ * Tagged template. Resolves against https://www.w3c.org.
```

Comment:
Why not use a regular function instead?

Reply:
There are heavy uses of template string so I think it's better to be a template string function.

Reply:
Hmm no, it can just be a wrapper without actual need for TSF...

Reply:
yeah, a function is probably fine. Doesn't look as cool tho 😎🤪.